conans/test/unittests/client/build/cpp_std_flags_test.py

```diff
@@ -214,6 +228,8 @@ def test_apple_clang_cppstd_defaults(self):
         self.assertEqual(_make_cppstd_default("apple-clang", "10"), "gnu98")
         self.assertEqual(_make_cppstd_default("apple-clang", "11"), "gnu98")
         self.assertEqual(_make_cppstd_default("apple-clang", "12"), "gnu98")
+        self.assertEqual(_make_cppstd_default("apple-clang", "13"), "gnu98")
+        self.assertEqual(_make_cppstd_default("apple-clang", "14"), "gnu98")
```

Comment:
```suggestion
```

Reply:
Same as above. I've put it in a separate commit ("v1.x")